packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js

```diff
@@ -14,6 +14,8 @@ export const overviewItem = {
     isBig: false,
     isTransparent: false,
     isCard: false,
+    isPercent: false,
```

Comment:
You could delete `isPercent` props and check it if `percentValue !== 0`

Reply:
![Screenshot 2023-12-21 at 14 23 33](https://github.com/hit-pay/orchid/assets/48382527/1f645e8d-cd93-44fb-a8a1-542e1b6d727e)
![Screenshot 2023-12-21 at 14 23 23](https://github.com/hit-pay/orchid/assets/48382527/0612cacb-97a1-4fe6-81e2-d31dd462ab23)

It is quite a same block with persentValue=0 and isPercent off/on

Reply:
Goos, will fix, thanks!